src/wiki/plugins/attachments/urls.py

```diff
@@ -1,4 +1,8 @@
 from django.conf.urls import url
+try:
+    from django.urls import re_path as url
+except ImportError:
+    from django.conf.urls import url
```

Comment:
Maybe it's best to move this logic to `wiki.core.compat` ?

Reply:
Ops, use `wiki.compat`.
Should move?